core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/domain/common/accessor/gizmo/AbstractReadWriteGizmoMemberAccessor.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.core.impl.domain.common.accessor.gizmo;
+
+public abstract class AbstractReadWriteGizmoMemberAccessor extends AbstractGizmoMemberAccessor {
+
+    @Override
+    public boolean supportSetter() {
+        return true;
+    }
+
+    @Override
+    public String getSpeedNote() {
+        return "Fast access with generated bytecode";
```

Comment:
Why was  this overwritten when `AbstractGizmoMemberAccessor` has the same code for this method?

Reply:
Good point, copy-paste error.